api/app/signals/apps/questionnaires/tests/test_models.py

```diff
@@ -122,6 +136,26 @@ def create_too_many_questions(graph_name='too_many'):
     return graph
 
 
+def create_illustrated_text():
```

Comment:
Not blocking but maybe the functions that create test data can be moved to a factory/utils file.